INTRODUCTION.md

```diff
@@ -1795,7 +1797,7 @@ The [Apache Kafka Implementation Proposals (KIPs)](https://cwiki.apache.org/conf
 | KIP-320 - Consumer: handle log truncation                                | 2.1.0 (partially implemented) / 2.2.0     | Not supported                                                                                 |
 | KIP-322 - DeleteTopics disabled error code                               | 2.1.0                                     | Supported                                                                                     |
 | KIP-339 - AdminAPI: incrementalAlterConfigs                              | 2.3.0                                     | Not supported                                                                                 |
-| KIP-341 - Update Sticky partition assignment data                        | 2.3.0                                     | Not supported                                                                                 |
+| KIP-341 - Update Sticky partition assignment data                        | 2.3.0                                     | Supported                                                                                     |
```

Comment:
if we decide to only do cooperative-sticky this would be not supported.

maybe a good reason to do 'sticky' as well as 'cooperative-sticky' is we get to cross off 2 more kips.

Reply:
we seem to have general agreement that skipping 'sticky' and going straight for 'cooperative-sticky' is fine.